conanfile.py

```diff
@@ -8,7 +8,7 @@
 
 class SISLConan(ConanFile):
     name = "sisl"
-    version = "9.0.1"
+    version = "9.0.2"
```

Comment:
I don't think we should bump the version of the library because of an added/modified unit test do you?

Reply:
I planned to add some change to the sg_list to replace vector to folly::small_vector, if it will not happen in this PR, I will remove this version change.

Reply:
added the change for small_vector